tools/define-compose-url.sh

```diff
@@ -2,19 +2,20 @@
 set -euo pipefail
 source /etc/os-release
 
-# This isn't needed when not running on RHEL
-if [[ $ID != rhel ]]; then
-  exit 0
+# This isn't needed when not running on RHEL or it's subscribed
+if [[ $ID != rhel ]] || sudo subscription-manager status; then
```

Comment:
How about the subscribed part? I'm wondering if we could end up in a situation where a subscribed test runner is trying to execute this script. Probably not, but in this case why do you need the check? 

If that's only because of ostree.sh IMO it makes sense to move the subscription check logic in ostree.sh b/c this file is used in other places as well.

Reply:
Yeah since it's only used in `ostree.sh` I moved the logic there, I'm not sure about whether to have the `return 0` or `exit 0` there... In case someone tries to sources it somewhere in the future his script could exit unexpectedly just like it was until now when `ostree.sh` was skipped on fedora and centos.

Reply:
`return 0` is in fact functionally correct. If that is not causing any other issues keep it. Strictly speaking the calling environment should be examining the exit code b/c in some situations COMPOSE_URL isn't going to be defined. However that's only used on RHEL test runners so I'm not expecting any errors.